pkg/modulereader/hcl_utils.go

```diff
@@ -82,15 +86,27 @@ func getHCLInfo(source string) (ModuleInfo, error) {
 	return ret, nil
 }
 
-// Transforms HCL type string into cty.Type
-func getCtyType(hclType string) (cty.Type, error) {
+// Transforms Terraform type string into cty.Type
```

Comment:
You s/HCL/Terraform/. How does this effect packer?

Reply:
Terraform type system is supperset of HCL one, by additional shorthand types `list` and `map`.
Packer is definetly a subset of TF one, no changes in behaviour.